### PR TITLE
k8smeetup-admin-ovs-networking-pr-2017-07-15

### PR DESCRIPTION
**Pull request by Tony-CCIE**
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.8 Features: set Milestone to `1.8` and Base Branch to `release-1.8`
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>
> NOTE: Please check the “Allow edits from maintainers” box (see image below) to
> [allow reviewers to fix problems](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) on your patch and speed up the review process.
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5356)

### CODE REVIEW

**Comment by Tony-CCIE** [comment]
i sign it

**Comment by markthink** [comment]
I approve of this PR.

File: cn/docs/admin/ovs-networking.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+![OVS Networking](/images/docs/ovs-networking.png)
+
+
+Kubernetes中Vagrant的设置如下：
+
+
+docker网桥被brctl生成的Linux网桥(kbr0)所代替，kbr0是具有256个地址空间的子网。总的来说，node会得到10.244.x.0/24的子网，docker上配置使用的网桥会代替默认docker0的网桥。
+
+
+另外，OVS网桥创建(obr0)，并将其作为端口添加到kbr0的网桥中。所有OVS网桥通过GRE隧道而连接所有的nodes。因此，每个node都有一个到其他nodes的出站GRE隧道。这个隧道没有必要是一个完整的网状物，但是越像网状结构越好。在网桥上开启STP(生成树)模式以防止环路的发生。
```
Comment (zhangxiaoyu-zidif) [inline cn/docs/admin/ovs-networking.md]: "所有OVS网桥通过GRE隧道而连接所有的nodes。"
读起来很绕口。
"所有OVS网桥通过GRE隧道去连接所有的nodes。"
或者
"所有OVS网桥通过GRE隧道连接所有的nodes。"

File: cn/docs/admin/ovs-networking.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+
+路由规则允许任何10.244.0.0/16通过与隧道相连的OVS网桥而到达目标。
```
Comment (zhangxiaoyu-zidif) [inline cn/docs/admin/ovs-networking.md]: 可以将“而”去掉吗。

File: cn/docs/admin/ovs-networking.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
```
Comment (zhangxiaoyu-zidif) [inline cn/docs/admin/ovs-networking.md]: 这里多余的空行可以去掉嘛？

Comment (Tony-CCIE) [reply cn/docs/admin/ovs-networking.md]: OK，我去修改一下

**Pull request merged**